astrology.lic

```diff
@@ -234,7 +234,16 @@ class Astrology
         bput('open my telescope', 'extend your telescope')
         return observe(body)
       end
-      result = bput('peer telescope', 'You peer aimlessly through your telescope', 'You see nothing regarding the future', 'Roundtime')
+      result = bput('peer telescope', 'You peer aimlessly through your telescope', 'You see nothing regarding the future', 'The pain is too much', 'Roundtime')
+        if result =~ /The pain is too much/
+          bput('stow telescope', 'you put', "That can't be picked up")
+          snapshot = Room.current.id
+          wait_for_script_to_complete('safe-room')
+          walk_to(snapshot)
```

Comment:
To use `walk_to` here, you need to add `common-travel` to `custom_require` and `include DRCT` above.